main/Main.hs

```diff
@@ -73,6 +76,7 @@ connectionWorker mainTid pool schema refDbStructure refIsWorkerOn = do
       connected <- connectingSucceeded pool
       when connected $ do
         result <- P.use pool $ do
+          fillSessionWithSettings settings
```

Comment:
This should go after the minimum PostgreSQL version validation, 6 lines below, otherwise it'll give an error on older versions because `unnest` with two args is only available on `>= 9.4`.